### ISSUE TITLE
Blue Mage Damage Scaling

### ISSUE THREAD

**Issue by DrfistMD** [EdenServer/community#1180]
Leveling BLU myself and speaking with other BLUs, the damage scaling of physical BLU spells seems to start dropping off dramatically after you start going past Tough mobs.  To the point where 2 sword swings with dual wield are generally outperforming a number of spells, but for whatever reason Bludgeon seems to be hit extra hard (possibly due to multi hit?).  In my own experience, and in screen shots below, even at level 36 Bludgeon is seen occasionally doing single digit damage, and its average damage per cast is so low that spells like Head Butt start to become on par with it, RNG depending.

So I went out and cast Foot Kick, Head Butt and Bludgeon on a **ton** of goblins of differing tiers, TW, EP, DC, T, VT and IT, and tracked the damage results for these casts.  My gear and /checkparam for these tests is screen shotted below.

The data from this tracking does really show BLU damage really starting to fall to really poor levels once mobs scale up.  But it also shows BLU spells doing a _ton_ of damage to TW mobs.  The accuracy of these physical spells also seems a bit lower than expected, as someone else has pointed in a bug.  There are also a handful of times where Bludgeon seems to be hitting for FAR lower than it ever should.  Even if assuming only 1 of the 3 hits of Bludgeon hit, the damage floor of 4 versus the average of 32 on IT mobs seems far too wide.  As I've reported in another bug that BLU physical spells are not properly reacting with SA, I'm wondering if they're being applied purely as magical attacks with the related resist chances and not feeding off physical accuracy and play properly.  I've also attached screenshots from my combat log directly showing these Bludgeons far below the expected.

I would also like to point that spells like Poison Breath and Digest with their "own" damage formulas do seem to be working exactly as intended, something just really does seem up with the physical BLU spells, to the point where they're lagging behind even just auto attacks at various level tiers.  I've tried to be as thorough as I could looking into this issue, if any other BLUs would like to chime in with their own experience, please do.

![stats](https://user-images.githubusercontent.com/46073012/53707577-8d690100-3dfd-11e9-9f1e-9cc27e395d5f.png)
![stats2](https://user-images.githubusercontent.com/46073012/53707738-33b50680-3dfe-11e9-84b9-612f305ecb89.PNG)
![sheet 1](https://user-images.githubusercontent.com/46073012/53707581-8f32c480-3dfd-11e9-9f9c-d50f116753b3.PNG)
![sheet 2](https://user-images.githubusercontent.com/46073012/53707582-8fcb5b00-3dfd-11e9-94b1-6ae745b92434.PNG)
![low bludgeon](https://user-images.githubusercontent.com/46073012/53707583-90fc8800-3dfd-11e9-952a-a2bb319ed141.PNG)

**Comment by DrfistMD** [comment]
Was curious if the SA issue was possibly related to this one, so I went out and tried again after the 3/11 patch against some IT gobs, and was still seeing some rather low damage.  A SA Smite of Rage however did pack quite a whallop by comparison however.
![SA Smite](https://user-images.githubusercontent.com/46073012/54166040-ab57e680-4439-11e9-82ae-ba1bf525e3e2.PNG)

**Comment by DrfistMD** [comment]
Spent a length of time in an XP party and observed that Head Butt was either doing around 60, 30 or 15 damage per cast.  This seems to line up with the magical resists observed for normal magic, however, BLU physical spells should not be subject to this resistance.  I believe this is what is driving the wide ranges and massive drop off in damage that's being observed on BLU physical spells as the spells are being both effected by normal attack v defense values on top of magical resists, when physical spells should be treated as purely physical.

![unknown](https://user-images.githubusercontent.com/46073012/54408567-9e94f600-46b8-11e9-8dd1-80a9b523e27d.png)

**Comment by BuffaloOnEden** [comment]
I should have reported this weeks ago, but blue physical spell accuracy is completely broken (in a bad way) on this server. Blue physical spell accuracy is supposed to be dependent upon blue magic skill and physical accuracy. Even with accuracy gear, capped blue magic, sole sushi, and full tp, bludgeon performs horribly on VT+. However, it is not just bludgeon: all physical blue spells have accuracy which does not appear to match that of physical accuracy. It appears to me that the metric for which blue physical accuracy is measured is based on stats which should not be relevant. Devs, please check it.

**Comment by bkesecker** [comment]
Forum post to collect community feedback posted.  https://www.edenxi.com/forums/viewtopic.php?f=1&t=1263

**Comment by DrfistMD** [comment]
After the adjustments on 4-8 I can confirm that BLU physical spell accuracy is far better now, but the spell damage still suffers from the same issue as I documented above.  New set of data below, but the differences between minimum, average and maximum on spells still seems way too high, even on single hit spells.  I did try testing a few higher level physical spells, and their clustering seems much tigher than the lower level spells.  But as far as things stand currently, the only spell really worth casting for efficient DPS is Head Butt for both damage per MP spent, as well as cool down on the spell itself, which is still out of whack.

Based on previous efforts, I did not do as large a sample size this time around as the results were very similar to the initial batch.

_(Yes my skills were capped for my level, if a Beetle used a defensive TP move, I did not record any further data from that mob)_

Also here's the full Google Doc where I've been tracking this info, if there's any dev interest in it.  It should be viewable, but not editable:
https://docs.google.com/spreadsheets/d/1HK5qmjm6K4H77xNKU61KPL2jrtu6aclSVLr0cEJUwak/edit#gid=803596612

![Stats](https://user-images.githubusercontent.com/46073012/56072639-edd94f80-5d66-11e9-9f8a-c539e20a044d.png)
![BLU data](https://user-images.githubusercontent.com/46073012/56072641-efa31300-5d66-11e9-848c-53fd4850680c.png)
